buildlib/tools/builds.sh

```diff
@@ -260,7 +262,7 @@ build_gcc() {
 		if az_module_load $GCC_MODULE
 		then
 			echo "==== Build with GCC compiler ($(gcc --version|head -1)) ===="
-			${WORKSPACE}/contrib/configure-devel --prefix=$ucx_inst
+			${WORKSPACE}/contrib/configure-devel CFLAGS=$cflags CXXFLAGS=$cflags --prefix=$ucx_inst
```

Comment:
BTW maybe use CFLAGS="$cflags" CXXFLAGS="$cflags" in case we pass multiple flags